lib/components/map/gtfs-rt-vehicle-overlay.js

```diff
@@ -1,237 +0,0 @@
-import { transit_realtime as transitRealtime } from 'gtfs-realtime-bindings'
-import L from 'leaflet'
-import TransitVehicleOverlay from '@opentripplanner/transit-vehicle-overlay'
-import PropTypes from 'prop-types'
-import React from 'react'
-import ReactDOMServer from 'react-dom/server'
-import { FeatureGroup, MapLayer, Marker, Polyline } from 'react-leaflet'
-import styled from 'styled-components'
-
-import { getModeFromRoute } from '../../util/viewer'
-
-export const VehicleShape = styled.span`
-  background-color: #${props => props.route?.route_color || '000000'};
-  border: 1px solid #000000;
-  border-radius: 50%;
-  color: #${props => props.route?.route_text_color || 'ffffff'};
-  display: block;
-  height: 24px;
-  left: -6px;
-  line-height: 24px;
-  margin-left: auto;
-  margin-right: auto;
-  position: relative;
-  text-align: center;
-  top: -6px;
-  width: 24px;
-`
-
-/**
- * Convert GTFS-rt entities to vehicle location from OTP-UI.
- * @param vehicle The GTFS-rt vehicle position entity to convert.
- * @param routes Optional GTFS-like routes list.
- */
-function gtfsRtToTransitVehicle (gtfsRtVehicle, routes) {
```

Comment:
Is there a replacement intended for this component?

Reply:
This component is needed for one of our deployments, so please reinstate or provide an alternate.

Reply:
Component is restored in 2ff5d6aa17ccc7adb6fad53a7637cf47e0f22ef2